__tests__/banner.spec.js

```diff
@@ -0,0 +1,18 @@
+import { mount } from '@vue/test-utils'

```

Comment:
It would be helpful if the test directory followed a similar patter to src so we can navigate easily. ie /\_\_tests\_\_/components/atoms/Banner.spec.js

Reply:
Good idea.